packages/p2p/stores/my-profile-store.js

```diff
@@ -0,0 +1,159 @@
+import { observable, action } from 'mobx';
+import { requestWS } from 'Utils/websocket';
+import { localize } from 'Components/i18next';
+import { textValidator } from 'Utils/validations';
+
+export default class MyProfileStore {
```

Comment:
Constructor should either inject `root_store` or it should extend a `BaseStore`.

```JS
constructor(root_store) {
    this.root_store = root_store;
}
```

Reply:
Yeap, sorry, missed that here.